check_release_assets.py

```diff
@@ -85,7 +102,7 @@ def get_expected_wheel_basename(project, version, abi, arch):
 
 
 def verify(project, expected, actual):
-    print('🔵 Project: {}'.format(project))
+    print('🔵 PyPI: {}'.format(project))
```

Comment:
This function is also called from L150. https://github.com/cupy/cupy-release-tools/pull/295/files#diff-bca831d24e956928fc7f3bf8bc599b9097cc39f205b135028a28433fd36db964R150

Reply:
Ah, nice catch!
```suggestion
    print('🔵 Project: {}'.format(project))
```